### PR TITLE
refactor: Connect to idOS on page load

### PR DESCRIPTION
**Pull request by shelegdmitriy**
This PR introduces a couple of changes:

1. The new tooltip about successful connection to idOS.
2. Added a message to the banner to inform users about preferred wallets to use the idOS feature.
3. Made banner visible by default.
4. Added ability to load idOS data on page load

Relates to: #493 #494

### CODE REVIEW

File: src/Settings/Identity/Index.jsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
-const [showBanner, setShowBanner] = useState(false);
+const [showBanner, setShowBanner] = useState(true);
 
 // const modalOpen = fractalModal?.alert ?? false;
 // const fractalModal = Storage.get("fractal-alert") ?? null;
```
Comment (calebjacob) [inline src/Settings/Identity/Index.jsx:4]: Does it make sense to get rid of these commented out lines?

File: src/Settings/Identity/Index.jsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
 
+const tooltipState = useMemo(() => props.showTooltip, [props.showTooltip]);
```
Comment (calebjacob) [inline src/Settings/Identity/Index.jsx:6]: This looks like a situation where `useMemo()` isn't providing any value.

File: src/Settings/Identity/Index.jsx [hunk 3]
```diff
@@ -30,6 +34,12 @@ const Icon = styled.i`
 
 const bannerToggle = useCallback(() => setShowBanner(!showBanner), [showBanner]);
 
+useEffect(() => {
+  if (!props.idosConnected && props.connectIdOS) {
+    props.connectIdOS()
+  }
+}, [props.idosConnected]);
```
Comment (calebjacob) [inline src/Settings/Identity/Index.jsx:41]: Looks like we might have forgot to add `props.connectIdOS` to the dependencies array?

Comment (shelegdmitriy) [reply src/Settings/Identity/Index.jsx]: Thanks for catch!

File: src/Settings/Identity/Index.jsx (hunk 1 above)
Comment (shelegdmitriy) [reply src/Settings/Identity/Index.jsx]: Yes. I fixed that logic related to modal

**Review by gabehamilton — APPROVED** [review]

**Pull request merged**